setup.py

```diff
@@ -5,7 +5,16 @@
 HERE = pathlib.Path(__file__).parent
 
 
-VERSION = "1.2.0"
+def get_version():
+    versionpath = pathlib.Path(HERE) / "obsws_python" / "version.py"
+    with open(versionpath) as f:
+        for line in f:
+            if line.startswith("version"):
+                versionstring = line.split('"')[1]
+    return versionstring
+
+
+VERSION = get_version()
```

Comment:
instead of using a function, can we import "version" variable from version.py and use it as follows

```suggestion
from obsws_python.version import version as __version__

VERSION = __version__
```